### ISSUE TITLE
Hey you forget a file !

### ISSUE THREAD

**Issue by Awea** [ruby-jokes/job_interview#11]
When I install this gem I getting an error like this : 

```
/Users/aweaoftheworld/.rvm/rubies/ruby-1.9.3-p194/lib/ruby/site_ruby/1.9.1/rubygems/custom_require.rb:36:in `require': cannot load such file -- /Users/aweaoftheworld/.rvm/gems/ruby-1.9.3-p194@bonjourgem/gems/job_interview-0.1.4/lib/job_interview/hello_world (LoadError)
```

So hello_world is messing after adding it all works :)

**Comment by semaperepelitsa** [comment]
This ruined my interview!!! :cry:

**Comment by canweriotnow** [comment]
@semaperepelitsa :laughing:

**Comment by canweriotnow** [comment]
I'll get a fix in ASAP. Wouldn't want to end any careers with my carelessness.

**Comment by Awea** [comment]
Hehe, thanks for this [gem](http://www.bonjourgem.com/post/27249888046/dimanche-cest-job-interview) :p

**Comment by canweriotnow** [comment]
Fixed by #15
